flow/layers/layer_tree.cc

```diff
@@ -155,6 +155,69 @@ void LayerTree::Paint(CompositorContext::ScopedFrame& frame,
   }
 }
 
+std::shared_ptr<const impeller::Picture> LayerTree::FlattenToImpellerPicture(
+    const SkRect& bounds,
+    const std::shared_ptr<TextureRegistry>& texture_registry) {
+#if IMPELLER_SUPPORTS_RENDERING
+  TRACE_EVENT0("flutter", "LayerTree::FlattenToImpellerPicture");
+
+  impeller::DlAiksCanvas canvas(bounds);
+
+  const FixedRefreshRateStopwatch unused_stopwatch;
+
+  LayerStateStack preroll_state_stack;
+  // No root surface transformation. So assume identity.
```

Comment:
I think Vulkan prerotation may eventually give us a root surface transformation.

Reply:
This comment is duplicated from `Flatten` below. But I guess we can handle that then? The idea is that this method would only ever be called on a `Scene` object that is not really attached to a particular surfacde.

Reply:
Yeah, no action needed, just a comment